src/Context.cpp

```diff
@@ -853,13 +854,8 @@ int Context::dispatch (std::string &out)
     // The command know whether they need a GC.
     if (c->needs_gc ())
     {
-      run_gc = config.getBoolean ("gc");
```

Comment:
This value is written but never read, so I removed it.

An early draft of this PR linked expiration to gc, so I was in this file making modifications. This simplification seemed worth keeping.